src/Components/CreateEventPage.js

```diff
@@ -1,12 +1,127 @@
 import React, { Component } from 'react';
+import { MDBInput, MDBContainer, MDBRow, MDBCol } from 'mdbreact';
+import './styles/CreateEventPage.css';
+import SubmitButton from './SubmitButton';
+import axios from "axios";
 
 class CreateEventPage extends Component {
     constructor(){
         super();
+        this.state = {
+            eventName: '',
+            exDescription: '',
+            inDescription: '',
+            address: '',
+            city: '',
+            link: ''
+        }
     }
+
     render() {
         return(
-            <h1>CreateEvent Page</h1>
+            <div className="app-page" id="create-page">
+                <div className="app-page-fill"></div>
+                <div className="app-main-section">
+                    <h1 className="app-page-header">Make Event</h1>
+                    <MDBContainer id="create-page-container">
+                        <form id="create-form">
+                            <MDBRow>
+                               <MDBCol size="3">
+                                    <h4>Event Name:</h4>
+                               </MDBCol>
+                               <MDBCol size="9">
+                                    <MDBInput
+                                    className="textField"
+                                    autoComplete="off"
+                                    type="text"
+                                    name="eventName"
+                                    onChange={(e) => this.setState({eventName: e.target.value})}
```

Comment:
Idk if @zhengwin can confirm, but I would recommend putting the key of any JSON in single quotes like `{'eventName' : e.target.value}` . 
This makes it unambiguously a literal string key. Whereas in the future if you happen to have a variable name called `eventName` it won't accidentally take the value of that variable as the key name. 
Happened to me once with node.js so I never did it since. You can keep it though if Jackson thinks it's not a problem in react.